util/measures/qcdr-measures-strata-names.json

```diff
@@ -6,7 +6,7 @@
   "ARCO13": ["antithrombotic", "anticoagulation", "antithrombotic2", "LDL"],
   "AHSQC9": ["overall", "email"],
   "AQI49": ["lysine", "mini", "redcell", "transfusion", "overall"],
-  "MOA1": ["back", "neck"],
+  "MOA 1": ["back", "neck"],
```

Comment:
@lamroger is it an issue to have a space in this measure id?

Reply:
(FYI, the space comes from the source data, it's not added on our end. I removed the space last time manually, but decided against re-removing it this time to make future changes less manual.)

Reply:
i think @lamroger removed the spaces when he parsed a csv elsewhere, so let's do that (remove whitespace)

Reply:
hm ok... done!